bool clear_sticky_faults
bool reset_total_energy
---
